pgx-tests/src/framework.rs

```diff
@@ -476,28 +460,29 @@ fn start_pg(loglines: LogLines) -> eyre::Result<String> {
 
     // start Postgres and monitor its stderr in the background
     // also notify the main thread when it's ready to accept connections
-    let (pgpid, session_id) = monitor_pg(command, command_str, loglines);
-
-    // add a shutdown hook so we can terminate it when the test framework exits
-    add_shutdown_hook(move || unsafe {
-        let message_string = std::ffi::CString::new(
-            format!("stopping postgres (pid={pgpid})\n").bold().blue().to_string(),
-        )
-        .unwrap();
-        libc::printf(message_string.as_ptr());
-        libc::kill(pgpid as libc::pid_t, libc::SIGTERM);
-    });
```

Comment:
No, `CString` did, but it's bad to pass a dynamic string as the first arg to `printf`, since if it has a `%` in it you'll probably crash. IN this case that should never happen, but it's good practice nonetheless.

Reply:
Ahhh okay.